instaclustr/resource_cluster_unit_test.go

```diff
@@ -501,11 +501,11 @@ func TestHasCassandraSizeChanges(t *testing.T) {
 func TestDoClusterResizeDefault(t *testing.T) {
 	err := doClusterResize(MockApiClient{
 		cluster: Cluster{
-			ID:         "CADENCE",
-			BundleType: "CADENCE",
+			ID:         "POSTGRESQL",
```

Comment:
**[QUESTION]** Why did you change the BundleType from `Cadence` to `PostgreSQL`?

Reply:
It is test the default behavior when users perform a resize on a NOT supported bundle type. Before is Cadence, now change to POSTGRESQL.